venues/MIDL.io/2019/Conference/python/create-review-invitations.py

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/python
+
+###############################################################################
+# ex. python create-invitations.py --conf MyConf.org/2017 --baseurl http://localhost:3000
+#       --username admin --password admin_pw
+#
+# To be run after submission due date to create review invitations for all the papers.
+# For each paper:
+# 1) create authorGroup (can see reviews, can't write a review)
+#           reviewer group (reviewers for this paper)
+#           and nonReviewerGroup (folks that aren't allowed to read the review at least not yet)
+# 2) create review invitation
+###############################################################################
+
+## Import statements
+import argparse
+import sys
+import openreview
+from openreview import tools
+import config
+
+## Argument handling
+parser = argparse.ArgumentParser()
+parser.add_argument('--baseurl', help="base url")
+parser.add_argument('--username')
+parser.add_argument('--password')
+
+args = parser.parse_args()
+
+## Initialize the client library with username and password
+client = openreview.Client(baseurl=args.baseurl, username=args.username, password=args.password)
+print("Connecting to "+client.baseurl)
+
+builder = openreview.conference.ConferenceBuilder(client)
+builder.set_conference_id(config.CONFERENCE_ID)
+builder.set_conference_name('Medical Imaging with Deep Learning')
+builder.set_conference_submission_name('Full_Submission')
+conference = builder.get_result()
+conference.set_authors()
+
+submissions = client.get_notes(invitation=config.CONFERENCE_ID+'/-/Full_Submission')
+for paper in submissions:
+    paper_num = str(paper.number)
+    paperinv = config.CONFERENCE_ID + '/-/Paper' + paper_num
+    paperGroup = config.CONFERENCE_ID + '/Paper' + paper_num
+    print(paper_num)
+    ## Reviewer group - people that can see the review invitation
+    reviewerGroup = paperGroup + '/Reviewers'
+    try:
+        client.get_group(id=reviewerGroup)
+        print("Found " + reviewerGroup)
+    except openreview.OpenReviewException as e:
+        # is group not found, then make groups and invites
+        print(e.args[0][0])
+        if e.args[0][0].startswith('Group Not Found'):
+            client.post_group(openreview.Group(
```

Comment:
I think these groups are created when we run tools.assign, right? you don't need to create them now.

Reply:
It does, but it doesn't include the AC for the paper as a reader.